wagtail/models/__init__.py

```diff
@@ -700,6 +700,7 @@ def preview_modes(self):
         will only have one display mode, but subclasses can override this -
         for example, a page containing a form might have a default view of the form,
         and a post-submission 'thank you' page.
+        Set to `[]` to completely disable previewing for this model.
```

Comment:
Docstrings are still interpreted as rst when they're pulled in to the docs, so this needs double-quoting:
```suggestion
        Set to ``[]`` to completely disable previewing for this model.
```

Reply:
Thank you - U didn't know that.